src/Api/bikeStationApi.ts

```diff
@@ -5,35 +5,31 @@ import axios from "axios";
 
 const station_url = BASE_URL + "stations/";
 
-export enum BikeStationState {
+export enum StationState {
     Working, Blocked,
 }
 
-export interface BikeStation {
+export interface Station {
     id: string;
-    state: BikeStationState;
+    state: StationState;
     name: string;
     bikes: Bike[];
 }
 
 export const postStation = async (stationName: string) => {
-    axios({
-        method: 'post',
-        url: station_url,
-        data: {
-            name: stationName,
-        }
-    });
+    return axios.post(station_url, {name: stationName}).then(r => axiosHandleResponse(r));
```

Comment:
Jeśli działa to spoko

Reply:
tak, po refaktorze sprawdziłem każdą z funkcjonalności